src/Root.tsx

```diff
@@ -9,6 +15,17 @@ import { createStore } from './store';
 // eslint-disable-next-line import/no-unassigned-import, import/extensions
 import './assets/fonts/fonts.css';
 
+const { chains, publicClient } = configureChains(
+  [linea, lineaTestnet],
+  [publicProvider()],
+);
+
+const wagmiConfig = createConfig({
+  autoConnect: true,
+  connectors: [new MetaMaskConnector({ chains })],
+  publicClient,
+});
```

Comment:
Can this be done in a separate file (`network.ts` or something), and imported here instead?

Reply:
Done